### ISSUE TITLE
Edit shortcut name after creation 

### ISSUE THREAD

**Issue by JeromeSivadier** [75py/FileShortcut#1]
Hi,

Really nice app! Would it be possible to add a shortcut name edition feature with a right click on the shortcut? On the latest Android versions it should just be a handler to add.

Thanks!

**Comment by 75py** [comment]
@Jerk31 Thank you!
It's very good idea. I'll try.

**Comment by 75py** [comment]
Implemented. The feature will release in next version. (soon)
<img src="https://user-images.githubusercontent.com/3153211/34601253-7593fcbc-f23e-11e7-8104-bacc1040a9a1.gif" width="270" height="480">

**Comment by 75py** [comment]
@Jerk31 Implemented and released.
Please try version 2.1.1 on Google Play.

**Comment by JeromeSivadier** [comment]
Really quick thx! It does not work though on my OnePlus phone (OxygenOS 4.5.15 based on Android 7.1.1). Did you implement it for Android Oreo and forward?

**Comment by 75py** [comment]
Oops!
I misunderstood, I thought that the App Shortcuts API is supported on Android Oreo or higher.
I'll fix it soon.

**Comment by 75py** [comment]
@Jerk31 As far as I searched, I found that the feature cannot work completely in Android 7.1.

The feature use those functions:
* ShortcutManagerCompat#requestPinShortcut(Context, ShortcutInfoCompat, IntentSender)
Use to create new shortcuts.
This funcation is the wrapper of ShortcutManager#requestPinShortcut().
> On API <= 25 it creates a legacy shortcut with the provided icon, label and intent. For newer APIs it will create a ShortcutInfo object which can be updated by the app.
* ShortcutManager#updateShortcuts(List<ShortcutInfo>)
Use to modify created shortcuts.
since API 25

updateShortcuts() can also use in Android 7.1.
But ShortcutManagerCompat#requestPinShortcut() creates a legacy shortcut in Android 7.1, so File Shortcut cannot update it.

I think that I can implement the similar feature by using "dynamic shortcuts" but the user interface will not be clear.
So I cannot implement your requested feature for Android 7.1.
